### PR TITLE
Ignore any calls to increment_metric as irrelevant

### PR DESCRIPTION
**Pull request by brrygrdn**
As title, we don't need to handle or set expectations on these calls.

### CODE REVIEW

**Review by landongrindheim — APPROVED** [review]

**Review by jakecoffman — APPROVED** [review]
Do we want to log them?

**Comment by brrygrdn** [comment]
@jakecoffman I was 50/50 on that, I was thinking of it mostly as 'I'm not sure anyone but us cares about instrumentation', but I guess their information is likely useful to have in the logs?

**Comment by jakecoffman** [comment]
I would only be concerned that someone is confused why isn't working since it's eating it quietly instead of eating it loudly.

**Comment by brrygrdn** [comment]
```bash
  proxy | 2023/04/06 14:05:59 [008] 200 http://host.docker.internal:57761/update_jobs/cli/update_dependency_list
  proxy | 2023/04/06 14:05:59 [009] POST http://host.docker.internal:57761/update_jobs/cli/increment_metric
{"data":{"metric":"updater.started","tags":{"operation":"Dependabot::Updater::Operations::UpdateAllVersions"}},"type":"increment_metric"}
  proxy | 2023/04/06 14:05:59 [009] 200 http://host.docker.internal:57761/update_jobs/cli/increment_metric
updater | 2023/04/06 14:05:59 INFO Starting update job for rsc/quote
```

@jakecoffman I've updated it to output the data, I think it makes more sense. I'll need to tweak the value of the `operation` in Core to avoid returning the full namespace 😅

**Comment by jakecoffman** [comment]
LGTM looks like it just needs a run through the Go formatter.

File: internal/server/api.go [hunk 1]
```diff
@@ -118,22 +118,19 @@ func (a *API) ServeHTTP(_ http.ResponseWriter, r *http.Request) {
 		a.pushError(err)
 	}
 
+	if kind == "increment_metric" {
+		// Let's just output the metrics data and stop
+		a.outputRequestData(kind, actual)
+		return
+	}
+
 	if err := a.pushResult(kind, actual); err != nil {
```
Comment (jakecoffman) [inline internal/server/api.go:127]: Oh you might need to skip the increment_metric here or it adds it to the assertions I think?

Comment (brrygrdn) [reply internal/server/api.go:127]: Ah, does the return on line 24 not break us out of the method?

Comment (jakecoffman) [reply internal/server/api.go:127]: Oh yes, I see you extracted the stdout output, perfect! 👍

**Pull request merged**